Packs/CrowdStrikeFalcon/Scripts/CsFalconLiftHostContainment/CsFalconLiftHostContainment.yml

```diff
@@ -0,0 +1,24 @@
+commonfields:
+  id: CsFalconLiftHostContainment
+  version: -1
+name: CsFalconLiftHostContainment
+script: ''
+type: python
+tags:
+  - Utilities
+comment: Lifts containment on the host, which returns its network communications to normal.
+enabled: true
+args:
+  - description: A comma-separated list of host agent IDs (AIDs) of the hosts to contain. Get an agent ID from a detection.
```

Comment:
A comma-separated list of host agent IDs (AIDs) to contain. Get agent IDs from detections.